### PR TITLE
[th2-2715] added util methods from store-common and removed dependency to store-common

### PR DESCRIPTION
**Pull request by andreydrobynin**
Related to https://github.com/th2-net/th2-common-j/pull/147, https://github.com/th2-net/th2-mstore/pull/23

### CODE REVIEW

**Review by Nikita-Smirnov-Exactpro — DISMISSED** [review]

File: src/main/java/com/exactpro/th2/estore/ProtoUtil.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        }
+        return builder.build();
+    }
+
+    public static StoredTestEventId toCradleEventID(EventIDOrBuilder protoEventID) {
+        return new StoredTestEventId(String.valueOf(protoEventID.getId()));
+    }
+
+    /**
+     * Converts protobuf event status to success Gradle status
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/estore/ProtoUtil.java:60]:
```suggestion
     * Converts protobuf event status to successful Gradle status
```

File: src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java [hunk 2]
```diff
@@ -49,26 +51,41 @@
...
+                    handle(delivery);
+                } catch (Exception e) {
+                    LOGGER.warn("Can not handle delivery from consumer = {}", tag, e);
+                }
+            }, ATTRIBUTES);
+            if (monitor != null) {
+                LOGGER.info("RabbitMQ subscribing was successful");
+            } else {
+                LOGGER.error("Can not find queues for subscribe");
+                throw new RuntimeException("Can not find queues for subscriber");
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java:84]:
```suggestion
                throw new RuntimeException("Cannot find queues for subscriber");
```

File: src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java [hunk 3]
```diff
@@ -103,30 +120,35 @@ public void handle(EventBatch eventBatch) {
...
     }
 
-    @Override
     public void dispose() {
-        super.dispose();
+        if (monitor != null) {
+            try {
+                monitor.unsubscribe();
+            } catch (Exception e) {
+                LOGGER.error("Can not unsubscribe from queues", e);
```
Comment (jean-michel-perli-sonsin) [inline src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java:128]:
```suggestion
                LOGGER.error("Cannot unsubscribe from queues", e);
```

**Review by jean-michel-perli-sonsin — DISMISSED** [review]

File: src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java (hunk 2 above)
Comment (andreydrobynin) [reply src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java]: done

File: src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java (hunk 3 above)
Comment (andreydrobynin) [reply src/main/java/com/exactpro/th2/estore/ReportRabbitMQEventStoreService.java]: done

**Comment by andreydrobynin** [comment]
@jean-michel-perli-sonsin, @Nikita-Smirnov-Exactpro, I changed another "can not" and "subscriber" -> "subscribe" (like in row above) in `ReportRabbitMQEventStoreService`, please see.

**Review by Nikita-Smirnov-Exactpro — APPROVED** [review]

**Review by jean-michel-perli-sonsin — APPROVED** [review]

**Pull request merged**